src/autoscaler/api/publicapiserver/route_helper.go

```diff
@@ -76,8 +76,8 @@ func parseParameter(r *http.Request, vars map[string]string) (*url.Values, error
 		return nil, fmt.Errorf("results-per-page must be greater than 0")
 	}
 	parameters := &url.Values{}
-	parameters.Add("start-time", startTime)
-	parameters.Add("end-time", endTime)
+	parameters.Add("start", startTime)
```

Comment:
Any test case for this?   When the `parameters`  is wrong previously,  I believe the related integration test case should fail ..  Or maybe we miss the corresponding integration test for this part.

Reply:
add test cases